CONTRIBUTING.md

```diff
@@ -16,15 +16,47 @@ The goals of this course (as outlined in the [syllabus](https://shawnrhoads.gith
 While I tried to incorporate a diverse curriculum to introduce undergraduate students to the basics of modeling in the context of social behavior, there is still so much I left out (and still much unknown in this growing field). That being said, I would absolutely love to improve and expand the content included in this Jupyter Book to utility beyond this course, especially with your help!
 
 Here are some ways you can help me accomplish this goal!
-- If you spot an error (e.g., typo, bug, inaccurate descriptions, etc.), please open a new issue on GitHub by clicking on the GitHub Icon in the top right corner on any page and selecting "open issue". Alternatively, you can [open a new issue directly through GitHub](https://github.com/shawnrhoads/gu-psyc-347/issues/new?title=Issue%20on%20page:%20*html%20&body=Your%20issue%20description%20here).
-- If I had inadvertently omitted credit for any content generated by others, please also let me know.
-- If you have an idea for a tutorial or a new module to include (especially related to the [growing list of readings](https://shawnrhoads.github.io/gu-psyc-347/module-00-03_Reading-List.html)), please either [open a new issue](https://github.com/shawnrhoads/gu-psyc-347/issues/new?title=Tutorial%20Idea:%20[INSERT%20TITLE]&body=Please%20suggest%20a%20tutorial%20idea%20here%20with%20any%20relevant%20resources.) and/or submit a pull request directly to the repository on GitHub.
+- If you spot an error (e.g., typo, bug, inaccurate descriptions, etc.), please open a new issue on GitHub by clicking on the GitHub Icon in the top right corner on any page and selecting "open issue". Alternatively, you can [open a new issue directly through GitHub](https://github.com/shawnrhoads/gu-psyc-347/issues/new?labels=bug&template=issue-template.yml).
```

Comment:
how does one access the different issue templates?

Reply:
the links should work once this PR is merged into master